bin/reth/src/commands/node/mod.rs

```diff
@@ -285,7 +279,7 @@ mod tests {
         let cmd =
             NodeCommand::try_parse_args_from(["reth", "--config", "my/path/to/reth.toml"]).unwrap();
         // always store reth.toml in the data dir, not the chain specific data dir
-        let data_dir = cmd.datadir.unwrap_or_chain_default(cmd.chain.chain);
+        let data_dir = cmd.datadir.unwrap_or_ch_default(cmd.chain.chain);
```

Comment:
what happened here? :)
```suggestion
        let data_dir = cmd.datadir.unwrap_or_chain_default(cmd.chain.chain);
```

Reply:
not sure....